iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -849,8 +847,9 @@ def create_poll_question(self, poll, poll_row, cur):
 
         poll_form_field = PollFormField.objects.create(
             page=poll, label=poll.title, field_type=field_type, choices=choices, admin_label=poll_row['short_name'])
+        cur.scroll(0, 'absolute')
```

Comment:
This line causes a crash for rke:

`psycopg2.ProgrammingError: scroll destination out of bounds`

Reply:
fixed